ui/src/app/layout/dashboard/dashboard.component.html

```diff
@@ -22,6 +22,9 @@ <h2 class="text-muted">Dashboard</h2>
                     <strong>Kafka Version:</strong>&nbsp;&nbsp;
                     <code
                         id="kafkaVersion">{{ (selectedEnvironment|async)?.authenticationMode === 'ccloud' ? 'Confluent Cloud' : (kafkaVersion | async) }}</code>
+                    <br>
+                    <strong>Galapagos Instance:</strong>&nbsp;&nbsp;
+                    <code>{{ instanceNameInfo | async }}</code>
```

Comment:
i am not sure, but since we now show the name in the header, this can be deleted from the dashboard i would say